tests/integration/conftest.py

```diff
@@ -960,12 +963,12 @@ def snowflake_feature_expected_tile_spec_dict_fixture():
     return expected_tile_spec
 
 
-@pytest.fixture
-def feature_manager(session):
-    """
-    Feature Manager fixture
-    """
-    return FeatureManager(session=session)
+# @pytest.fixture
+# def feature_manager(session):
+#     """
+#     Feature Manager fixture
+#     """
+#     return FeatureManager(session=session)
```

Comment:
Should we remove this? Or this is intended?

Reply:
It should be removed. Will fix.